main.tf

```diff
@@ -1,40 +1,30 @@
 # Define composite variables for resources
 module "label" {
-  source    = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.1.0"
+  source    = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.2.1"
   namespace = "${var.namespace}"
   name      = "${var.name}"
   stage     = "${var.stage}"
 }
 
 module "subnets" {
-  source = "git::https://github.com/cloudposse/tf_subnets.git?ref=tags/0.1.6"
-
+  source             = "git::https://github.com/cloudposse/terraform-aws-dynamic-subnets.git?ref=tags/0.2.3"
   availability_zones = "${var.availability_zones}"
   namespace          = "${var.namespace}"
   name               = "${var.name}"
   stage              = "${var.stage}"
   region             = "${var.region}"
   vpc_id             = "${aws_vpc.default.id}"
   cidr_block         = "${aws_vpc.default.cidr_block}"
+  igw_id             = "${var.igw_id}"
```

Comment:
It won't work, because `aws_internet_gateway` is being created in this module.
See [link](https://github.com/cloudposse/terraform-aws-vpc/pull/6/files#diff-7a370d8342e7203b805911c92454f0f4R27)